zerver/worker/queue_processors.py

```diff
@@ -451,10 +451,7 @@ def consume(self, data: Mapping[str, Any]) -> None:
         logger.info(
             "Sending invitation for realm %s to %s", referrer.realm.string_id, invitee.email
         )
-        if "email_language" in data:
-            email_language = data["email_language"]
-        else:
-            email_language = referrer.realm.default_language
+        email_language = data.get("email_language", referrer.realm.default_language)
```

Comment:
This one might also be better not simplified -- it's valuable to have and asset the presence of test coverage on this sort of conditional in this code path, since it's the kind of corner case that can tend to be broken in refactoring.